db/migrate/20210804184031_add_is_hidden_to_section_due_date.rb

```diff
@@ -0,0 +1,5 @@
+class AddIsHiddenToSectionDueDate < ActiveRecord::Migration[6.1]
+  def change
```

Comment:
Another thing we should change in the database is the name of the table itself. From section_due_dates to assessment_section_properties (or whatever it was we decided).

This will require updates to the model name too but that should be easy to fix with some find/replace